app/javascript/packs/application.js

```diff
@@ -7,6 +7,9 @@ import Rails from "@rails/ujs"
 import Turbolinks from "turbolinks"
 import * as ActiveStorage from "@rails/activestorage"
 import "channels"
+import 'bootstrap'
+require("channels")
+
```

Comment:
Please remove the lines 11 y 12 as well. The `channels` lib was already imported above.